pallet/assets-ext/rpc/runtime-api/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "pallet-assets-ext-rpc-runtime-api"
+version = "2.2.0"
+authors = ["The Root Network Team"]
+edition = "2021"
+license = "GPL-3.0"
+
+[dependencies]
+codec = { version = "3.0.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
+sp-api = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.30" }
+sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.30" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.30" }
+pallet-assets-ext = { path = "../../../assets-ext", default-features = false }
+seed-primitives = { path = "../../../../primitives", default-features = false}
```

nit: seperate substrate imports from local imports with empty line
```suggestion
sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.30" }

pallet-assets-ext = { path = "../../../assets-ext", default-features = false }
seed-primitives = { path = "../../../../primitives", default-features = false}
```